cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -534,16 +534,20 @@ describe("Get-into-teaching - teachet training adviser flow", () => {
 		cy.selectStage("Equivalent-Secondary");
 		cy.whichSubjectAreYouInterestedInTeaching("Equivalent-Computing");
 		cy.whenDoYouWantToStartYourTeacherTraining("Equivalent-2021");
-		cy.get("#equivalent_date_of_birth_date_of_birth_3i").type("27");
-		cy.get("#equivalent_date_of_birth_date_of_birth_2i").type("07");
-		cy.get("#equivalent_date_of_birth_date_of_birth_1i").type("1983");
-		cy.clickOnContinueButton();
-		cy.get("#equivalent-uk-or-overseas-uk-or-overseas-overseas-field").click();
+		cy.get("#teacher_training_adviser_steps_date_of_birth_date_of_birth_3i").type("27");
+		cy.get("#teacher_training_adviser_steps_date_of_birth_date_of_birth_2i").type("07");
+		cy.get("#teacher_training_adviser_steps_date_of_birth_date_of_birth_1i").type("1983");
 		cy.clickOnContinueButton();
-		cy.get("#equivalent-overseas-country-country-id-field").select("Austria");
+		cy.get("#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-overseas-field").click();
 		cy.clickOnContinueButton();
-		cy.get("#equivalent-overseas-candidate-telephone-field").type("0125234490");
+		cy.get("#teacher-training-adviser-steps-overseas-country-country-id-field").select("Austria");
 		cy.clickOnContinueButton();
+		cy.get("#teacher-training-adviser-steps-overseas-telephone-telephone-field").type("0125234490");
+    cy.clickOnContinueButton();
+		cy.get(".govuk-fieldset__heading")
+			.should("exist")
```

Comment:
This is a test change - as far as I can tell we should be showing the callback step for equivalent candidates (overseas in this case)